homeassistant/components/zeroconf/__init__.py

```diff
@@ -69,7 +75,12 @@ def zeroconf_hass_start(_event):
         Wait till started or otherwise HTTP is not up and running.
         """
         _LOGGER.info("Starting Zeroconf broadcast")
-        zeroconf.register_service(info)
+        try:
+            zeroconf.register_service(info)
+        except NonUniqueNameException:
+            _LOGGER.warning(
```

Comment:
This should be an error. As it means zeroconf won't run.

Reply:
Done